### PR TITLE
Harmonize qe-sap-deployment config.yaml

### PR DESCRIPTION
**Pull request by mpagot**
Made conf.yaml templates used by Trento and qesap regressio more consisten.
 - Copyright file header
 - Hardcoded provider: as in any case each config.yaml is specific to one CPS
 - Remove Ansible variables
 - String quotations

Related ticket: [TEAM-8049](https://jira.suse.com/browse/TEAM-8049)

Verification run:
 -  Results for sle-15-SP4-Qesap-Azure-Byos-x86_64-BuildLATEST_AZURE_SLE15_4_BYOS-qesap_azure_sapconf_test http://openqaworker15.qa.suse.cz/tests/200686 :green_circle: 
 -  Results for sle-15-SP4-Qesap-Azure-Byos-x86_64-BuildLATEST_AZURE_SLE15_4_BYOS-qesap_azure_saptune_test http://openqaworker15.qa.suse.cz/tests/200687 :green_circle: 
 -  Results for sle-15-SP5-Qesap-Azure-Byos-x86_64-BuildLATEST_AZURE_SLE15_4_BYOS-qesap_azure_saptune_test  `os_image_uri:`  http://openqaworker15.qa.suse.cz/tests/200698 :green_circle: using image uploaded by http://openqaworker15.qa.suse.cz/tests/200692
 - Result for sle-15-SP4-Qesap-Aws-Byos-x86_64-qesap_aws_sapconf_test http://openqaworker15.qa.suse.cz/tests/200694  :green_circle: 
 - Result for sle-15-SP4-Qesap-Aws-Byos-x86_64-qesap_aws_saptune_test http://openqaworker15.qa.suse.cz/tests/200695 :green_circle: 
 - Trento http://openqaworker15.qa.suse.cz/tests/200697 http://openqaworker15.qa.suse.cz/tests/200720 :green_circle: (the qe-sap-deployment part is OK)

### CODE REVIEW

File: tests/sles4sap/qesapdeployment/configure.pm [hunk 1]
```diff
@@ -22,7 +22,6 @@ sub run {
     my $qesap_provider = lc get_required_var('PUBLIC_CLOUD_PROVIDER');
 
     my %variables;
-    $variables{PROVIDER} = $qesap_provider;
```
Comment (alvarocarvajald) [inline tests/sles4sap/qesapdeployment/configure.pm:25]: Is this really needed?

**Review by alvarocarvajald — APPROVED** [review]
LGTM

File: tests/sles4sap/qesapdeployment/configure.pm (hunk 1 above)
Comment (mpagot) [reply tests/sles4sap/qesapdeployment/configure.pm:25]: It is no more needed line of code as the config.yaml no more has PROVIDER  as configurable variable. So I deleted the no more needed line of code.
Having it in the code is not harmful but just no more needed.

**Review by BillAnastasiadis — APPROVED** [review]
LGTM

File: data/sles4sap/qe_sap_deployment/qesap_azure.yaml [hunk 2]
```diff
@@ -1,35 +1,40 @@
...
-    vnet_address_range: "%VNET_ADDRESS_RANGE%"
-    subnet_address_range: "%SUBNET_ADDRESS_RANGE%"
-
+    az_region: '%REGION%'
+    deployment_name: '%DEPLOYMENTNAME%'
+    os_image: '%OS_VER%'
+    public_key: '%SSH_KEY_PUB%'
+    hana_remote_python: '%ANSIBLE_REMOTE_PYTHON%'
+    iscsi_remote_python: '%ANSIBLE_REMOTE_PYTHON%'
+    hana_instancetype: '%HANA_INSTANCE_TYPE%'
```
Comment (mpagot) [inline data/sles4sap/qe_sap_deployment/qesap_azure.yaml:17]: `qe-sap-deployment/terraform/azure` does not have variable like `hana_instancetype`. It is an AWS only variable.

File: data/sles4sap/qe_sap_deployment/qesap_azure_uri.yaml [hunk 3]
```diff
@@ -1,35 +1,41 @@
...
-    hana_remote_python: "%ANSIBLE_REMOTE_PYTHON%"
-    iscsi_remote_python: "%ANSIBLE_REMOTE_PYTHON%"
+    az_region: '%REGION%'
+    deployment_name: '%DEPLOYMENTNAME%'
+    public_key: '%SSH_KEY_PUB%'
+    hana_remote_python: '%ANSIBLE_REMOTE_PYTHON%'
+    iscsi_remote_python: '%ANSIBLE_REMOTE_PYTHON%'
     os_image_uri: 'https://%STORAGE_ACCOUNT_NAME%.blob.core.windows.net/sle-images/%OS_VER%'
-    hana_instancetype: "%HANA_INSTANCE_TYPE%"
+    hana_instancetype: '%HANA_INSTANCE_TYPE%'
```
Comment (mpagot) [inline data/sles4sap/qe_sap_deployment/qesap_azure_uri.yaml:17]: `qe-sap-deployment/terraform/azure` does not have variable like `hana_instancetype`. It is an AWS only variable.

**Pull request merged**